next/docs/common/entities.ts

```diff
@@ -12,6 +12,8 @@ export enum NavigationKey {
   CONSORTIA = "consortia",
   OVERVIEW = "overview",
   TEAM = "team",
+  FAQ = "faq",
```

Comment:
@esthermcdade can you check whether or not you get an error on the FAQ - it should be arranged alphabetically... let me know if VSC is showing you an error here please?